README.md

```diff
@@ -1,2 +1,55 @@
 # Shift_Resolve
-Shift catalog of Operators for Resolve
+
+## Description
+
+Shift catalog of Operators for Davinci Resolve.
+This catalog include multiple Shift operators to manage your Resolve project; the media pool elements like folders, videos and images; the timelines and the timeline items; the takes; and much more.
+
+## Installation
+
+To add this catalog to Shift, you can open the Catalog Manager `File > Catalog Manager` and add the catalog like and user Catalog. Just click on the `Add Catalog` button and select the shift_resolve.py file.
```

Comment:
```suggestion
To add this catalog to Shift, open the Catalog Manager by navigating to `File > Catalog Manager` in the Shift top menu and click on the "Add Catalog" button in the right top corner. This will open a file browser, allowing you to select the *shift_resolve.py* file. After adding the catalog, ensure that you apply the changes in the Catalog Manager to make them effective in Shift.
```